community/modules/file-system/cloud-storage-bucket/outputs.tf

```diff
@@ -62,3 +62,14 @@ output "gcs_bucket_path" {
     google_storage_bucket.bucket
   ]
 }
+
+output "gcs_bucket_name" {
+  description = "Bucket name."
+  # cannot use resource attribute, will cause lookup failure in startup-script
+  value = local.name
```

Comment:
nit. Does it cause lookup failure even with `depends_on` set ?

Reply:
Yes and No. IIUC there is some weird behavior because of this line:
https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/3abddcfbd245b0e6747917a4e55b30658414ffd7/modules/scripts/startup-script/main.tf#L130

If bucket name is not know at apply time then startup-script does not know if it is creating a bucket or not, and the bucket name gets written into vm and slurm startup scripts.

But this is a new variable that is not being used in such a convoluted way, so I agree to start out using it the right way.